Jenkinsfile

```diff
@@ -167,14 +170,33 @@ pipeline {
           environment {
             platform = 'centos6'
           }
-          steps {
-            sh 'rm -f apache-couchdb-*.tar.gz'
-            unstash 'tarball'
-            sh( script: build_script )
-          } // steps
+          stages {
```

Comment:
stages within a stage! :)

Reply:
Yeah that's a new feature :) I only did it to provide a little extra visibility into the current progress of each parallel build.